proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -58,24 +58,6 @@ service Service {
       body: "*"
     };
   }
-  // TxEncodeAmino encodes an Amino transaction from JSON to encoded bytes.
```

Comment:
I think I agree; we can still encode to amino JSON via the encoder in tx or is this talking about a different amino format? (I remember something about a binary amino format).

Instead maybe keep the messages/services and have the decode service return an error since this is what we're dropping support for.

Reply:
technically this doesn't work for modules that dont registerlegacyamino, so we have a bit of a discrepancy on support here. I can look at adding back the encode but decode id just remove.